bigtop.mk

```diff
@@ -293,10 +293,10 @@ IMPALA_NAME=impala
 IMPALA_RELNOTES_NAME=Impala
 IMPALA_PKG_NAME=impala
 IMPALA_BASE_VERSION=2.2.0
-IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.3.0
+IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.4.0
 IMPALA_RELEASE_VERSION=1
```

Comment:
Whoops looks like I didn't set this to `0.rc1` for the release candidate build. I hope that doesn't mess things up.

For upgrades from rc1 to this final build there's a chance we'll have to make this `2`. But that would only help us internally, since we didn't publicly release rc1 or anything.

What do you think?